cmd/config.go

```diff
@@ -204,6 +207,39 @@ type TLSConfig struct {
 	CACertFile *string
 }
 
+// Load reads and parses the certificates and key listed in the TLSConfig, and
+// returns a *tls.Config suitable for either client or server use.
+func (t TLSConfig) Load() (*tls.Config, error) {
```

Comment:
Can you write a brief unit test for this function? It might be easier if the `fmt.Errorf()`'s were replaced by constants where they don't include format strings.